Warehouse.Client/warehouse-client/src/app/services/data/data.service.ts

```diff
@@ -0,0 +1,37 @@
+import {Inject, Injectable, Optional} from '@angular/core';
+import {HttpClient, HttpHeaders} from "@angular/common/http";
+
+@Injectable({
+  providedIn: 'root'
+})
+export class DataService {
+  url!: string;
+
+  constructor(private http: HttpClient) {
+  }
+
+  options = {
+    headers: new HttpHeaders()
+      .set('Authorization', `Bearer ${localStorage.getItem('jwtToken')}`)
+      .set('Content-Type', 'application/json')
+      .set('Access-Control-Allow-Origin', '')
```

Comment:
Access-Control-Allow-Origin header should be sent from the BE. Check CORS settings